pack/tde/sfk_encounter.json

```diff
@@ -0,0 +1,873 @@
+		"hidden": true,
+		"name": "Endless Secrets",
+		"pack_code": "sfk",
+		"position": 129,
+		"quantity": 1,
+		"text": "You have uncovered a Sign of the Gods <i>(place 1 resource on the scenario reference card to indicate this)</i>. For the remainder of the scenario, resources on the scenario reference card represent Signs of the Gods the investigators have uncovered. Your quest is to uncover as many of these signs as you can in order to discern the location of Kadath.\nFlip this card back over.",
+		"type_code": "story"
+	},
+	{
+		"clues": 1,
+		"code": "06130",
+		"encounter_code": "the_search_for_kadath",
+		"encounter_position": 12,
+		"faction_code": "mythos",
+		"flavor": "It is said that the history of this land is written upon the famed cylinders of this ancient city: great brick pillars that rise into the sky, studied by sages, sorcerers, and scholars alike. Perhaps here you can find answers.",
+		"illustrator": "Hai Hoang",
+		"name": "Kadatheron",
+		"pack_code": "sfk",
+		"position": 130,
+		"quantity": 1,
+		"shroud": 5,
+		"text": "Veiled.\n[action] [action] [action]: Draw 5 cards.",
+		"traits": "Mnar. Ancient. City. Port.",
+		"type_code": "location"
+	},
+	{
+		"back_link": "06130",
+		"code": "06130b",
+		"faction_code": "mythos",
+		"flavor": "You study the pillars within Kadatheron for many hours, losing yourself in the vast, storied histories of the Dreamlands. What was first a focused search for the whereabouts of the castle of the gods soon becomes an obsession with all things inscribed upon the cylinders. Hours, days, weeks; who knows how much time you lose to your research. Though you learn much, the time you spend here is a boon to those who pursue you. You are only able to pry yourself free from the cylinders when you feel the ominous gaze of something sinister watching you from afar...",
+		"hidden": true,
+		"name": "Cylinders of Kadatheron",
+		"pack_code": "sfk",
+		"position": 130,
+		"quantity": 1,
+		"text": "You have uncovered a Sign of the Gods <i>(place 1 resource on the scenario reference card)</i>.\nShuffle 1 set-aside copy of Tenebrous Nightgaunt into the encounter deck, if able.\nFlip this card back over.",
+		"type_code": "story"
+	},
+	{
+		"clues": 1,
+		"code": "06131",
+		"encounter_code": "the_search_for_kadath",
+		"encounter_position": 13,
+		"faction_code": "mythos",
+		"flavor": "\"Not even the mines of precious metal remained. DOOM had come to Sarnath.\"\n– H. P. Lovecraft, <u>The Doom That Came to Sarnath</u>",
```

Comment:
NB \n- to \n -

Reply:
I think for this story we do put it in speechmarks rather than <u> - so change to \" to mirror style on card.